Index: msv1/src/main/java/com/example/msv/entity/MapData.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- msv1/src/main/java/com/example/msv/entity/MapData.java	(date 1561217151000)
+++ msv1/src/main/java/com/example/msv/entity/MapData.java	(date 1561217151000)
@@ -0,0 +1,21 @@
+package com.example.msv.entity;
+
+import lombok.Data;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * create by yongli on 2019-06-08 19:50
+ */
+@Data
+public class MapData {
+
+    public MapData(Map<String, String> map, List<String> ls) {
+        this.map = map;
+        this.ls = ls;
+    }
+
+    Map<String,String> map;
+    List<String> ls ;
+}
